src/theme/src/components/pane.js

```diff
@@ -0,0 +1,65 @@
+function getHoverElevationStyles(theme, hoverElevation) {
+  if (!Number.isInteger(hoverElevation)) return {}
+
+  return {
+    transform: 'translateY(-2px)',
+    boxShadow: theme.elevations[hoverElevation]
+  }
+}
+
+function getActiveElevationStyles(theme, activeElevation) {
+  if (!Number.isInteger(activeElevation)) return {}
+
+  return {
+    transform: 'translateY(-1px)',
+    boxShadow: theme.elevations[activeElevation]
+  }
+}
+
+function borderProperty(theme, { value, border }) {
+  if (
+    Object.prototype.hasOwnProperty.call(
+      theme.colors.border,
+      value
+    )
+  ) {
+    return `1px solid ${theme.colors.border[value]}`
+  }
+
+  if (value === true) {
+    return `1px solid ${theme.colors.border.default}`
+  }
+
+  if (value === false) {
+    return null
+  }
+
+  if (Object.prototype.hasOwnProperty.call(theme.colors.border, border)) {
+    return `1px solid ${theme.colors.border[border]}`
+  }
+
+  if (border === true) {
+    return `1px solid ${theme.colors.border.default}`
+  }
+
+  return value
+}
+
+export default function getPaneStyles(theme, props) {
```

Comment:
I ended up going down this path because it makes the most sense for now and keeps _all the styles_ in one spot. cc @colinlohner @akleiner2

Reply:
Spicy. Sounds good though!